### PR TITLE
General tidy up of XUI suite

### PR DESCRIPTION
**Pull request by jonsoloway-hmcts**
### JIRA link (if applicable) ###

N/A

### Change description ###

General tidy up of the XUI performance suite

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/simulations/ExUI.scala [hunk 1]
```diff
@@ -27,23 +27,18 @@ class ExUI extends Simulation {
...
-  val MChttpProtocol = Environment.HttpProtocol
-		//.proxy(Proxy("proxyout.reform.hmcts.net", 8080).httpsPort(8080))
-		.baseUrl(BaseURL).inferHtmlResources().silentResources
-		//.baseUrl("https://xui-webapp-perftest.service.core-compute-perftest.internal")
-		//.baseUrl("https://ccd-case-management-web-perftest.service.core-compute-perftest.internal")
+  val MChttpProtocol = http
+		.baseUrl(BaseURL)
+		.inferHtmlResources()
+		.silentResources
+    //.userAgentHeader("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36")
```
Comment (jmcadam) [inline src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/simulations/ExUI.scala:40]: Can we get rid of this if it's still commented out?

File: src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/Feeders.scala [hunk 2]
```diff
@@ -1,118 +1,14 @@
...
-    "-user"
-  })));
-
-  val IACCreateDataFeeder = Iterator.continually(Map("service" -> ({
-    "IACC"
-  }),
-    "SignoutNumber" -> ({
-      "280"
-    })
+  val orgName = "pforgdiv-" + Common.randomString(5)
```
Comment (jmcadam) [inline src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/Feeders.scala:7]: Need to confirm if everything will run ok with the feeder file tidied up, may even be worth moving these objects to the Common file instead and getting rid of the feeder file?

**Review by jmcadam — CHANGES_REQUESTED** [review]

File: src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/simulations/ExUI.scala (hunk 1 above)
Comment (jonsoloway-hmcts) [reply src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/simulations/ExUI.scala:40]: I commented this out to test it. Will remove it.

**Review by jmcadam — APPROVED** [review]
👍

**Pull request merged**